### ISSUE TITLE
No maintenance/new features in Marzipano --> Community take over?

### ISSUE THREAD

**Issue by joostvp** [google/marzipano#416]
Hi,

I really love the Marzipano tool and its flexibility. Since support feels "kinda dead" I would like to propose that Marzipano will be maintained by other people in the community. Is that possible @tjgq ?? 

thanks in advance, can we please contact me at joostvanpoppel[at]gmail.com?

thanks,
Joost

**Comment by trevorparscal** [comment]
I agree that this appears to be the case. Count me in on being a maintainer - we could fork it to another repo so we have control.

**Comment by jordyno** [comment]
That would help. I also see @trevorparscal's PR hasn't been added yet.

**Comment by Maverick87Shaka** [comment]
I'm looking for a good 360 tour maker. so far marzipano has the best final user experience, but the actual limit of the "tool" that cannot save,load and edit past project is literally a blocking option for big tour.

If anyone want start over, I'll be very happy to see has first feature the ability to save a project, and edit without recreating from the scratch.
